new-frontiers-14/frontier-station-14
# Better dock signage

## About the PR



Adds two posters, one saying "DOCK" and one saying "BUS" that replace the existing "EVAC" signs.

Removes some delays from static directional signs.

## Why / Balance


Looks better.

## How to test


1. Spawn, warp to Frontier, Trade Outpost, buy a few ships, check that the signs make sense.

## Media


Dock sign.
![image](https://github.com/user-attachments/assets/0ea2749d-d062-43bc-a694-f9f76d4afffd)

Bus sign.
![image](https://github.com/user-attachments/assets/8f2ab4fd-3391-4ddd-9c96-e06ff794fcc5)

## Requirements


## Breaking changes


**Changelog**



:cl:
- fix: Dock and bus signs should be correct now instead of appearing as "EVAC" signs.
